### DISCUSSION TITLE
Control LLDP configuration at interface level

### DISCUSSION THREAD

**Discussion by raviteja-b** [lldpd/lldpd#694] [Q&A]
I have lldpd running on my system with privilege separation disabled 
our system has two ethernet  interfaces and need to control LLDP on each interface at runtime.
we need to disable lldp on interface 1 while interface 2 has lldp turned on.

I tried adding interface configuration to  lldpd.conf file but nothing works 

can you please share lldp configuration to use in  lldpd config file

**Comment by raviteja-b** [comment]
@vincentbernat can you please share details on how to configure lldp for different interfaces with privilege separation disabled

**Reply by raviteja-b** [reply]
> @vincentbernat can you please share details on how to configure lldp for different interfaces with privilege separation disabled

I see lldp packets always sent out on ethernet interfaces when lldpd is running.
can you please let me know how to configure interfaces to disable/enable lldp packet emit

**Reply by vincentbernat** [reply]
Check the documentation: https://lldpd.github.io/usage.html.

**Reply by raviteja-b** [reply]
I have checked these options @vincentbernat 
does these options work if I pass through /etc/default/lldpd.conf ??
